draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -365,10 +369,12 @@ coordinate.
 ### Random Oracle {#term-rom}
 
 It is often the case that the output of the encoding function {{term-encoding}}
-should be distributed uniformly at random on the elliptic curve. That is, there
-is no discernible relation existing between outputs that can be computed based
-on the inputs. In practice, this requirement stems from needing a random oracle
-which outputs elliptic curve points:  one way to construct this is by first
+should be (a) distributed uniformly at random on the elliptic curve and (b) invertible.
```

Comment:
The invertible part generally doesn't hold for random oracle constructions.

Reply:
Oops. I meant to say non-invertible!